shared/message/vision/Ball.proto

```diff
@@ -34,20 +34,24 @@ message Ball {
     }
     message Measurement {
         MeasurementType type = 1;
-        // Vector from camera to ball in camera coordinates
+        /// Vector from camera to ball in camera coordinates
         fvec3 srBCc      = 2;  // Spherical Reciprocal Coordinates (1/distance, phi, theta)
         fmat3 covariance = 3;
     }
-    Cone                 cone           = 1;
-    repeated Measurement measurements   = 2;
-    fvec2                screen_angular = 3;
-    fvec2                angular_size   = 4;
-    fvec4                colour         = 5;
+    /// Cone describing the ball from the camera's perspective
+    Cone                 cone         = 1;
+    repeated Measurement measurements = 2;
+    /// Angular positions from the camera
+    fvec2 screen_angular = 3;
+    fvec2 angular_size   = 4;
+    fvec4 colour         = 5;
```

Comment:
If you're going to comment some of these I think you should comment the rest, the way it looks at the moment it seems that the comments apply to the things below them since there is no whitespace

Reply:
Roger that. I have added whitespace before `angular_size` (it seems to be assigned a bounding-box pixel {x, y} size at `HeadBehaviourSoccer:634`?? not sure if I should make an issue), and a few others I don't have comments for :+1: